devicetypes/smartthings/zwave-dimmer-switch-generic.src/README.md

```diff
@@ -0,0 +1,39 @@
+# Z-wave Dimmer
+
+
+
+Works with: 
+
+* [Leviton Plug-in Lamp Dimmer Module (DZPD3-1LW)](https://support.smartthings.com/hc/en-us/articles/206171053-How-to-connect-Leviton-Z-Wave-devices)
```

Comment:
Should this link to? http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZPD3-1LW
Since the support page is included below.